cogs/cogs.py

```diff
@@ -1,12 +1,14 @@
 from os import listdir
-from discord.ext.commands import slash_command
-from discord.ext import commands
+
+import discord
 from discord import Option
 from discord.ext.bridge import Bot
+from discord.ext.commands import slash_command
+
 from utilities.data import get_data
```

Comment:
Again, group the remaining imports together

Reply:
Done to seperate third party imports (discord) from local imports (utilities.data) and built-in imports (asyncio)